impeller/fixtures/runtime_stage_position.frag

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <flutter/runtime_effect.glsl>
+
+uniform vec4 ltrb;
+
+out vec4 frag_color;
+
+// Output solid red if frag position is within LTRB rectangle.
+void main() {
```

Comment:
This only outputs red if the rendered coordiantes match the expected coordinates.